Quadriga/src/main/java/edu/asu/spring/quadriga/accesschecks/impl/ProjectSecurityChecker.java

```diff
@@ -43,8 +42,7 @@
      */
     @Override
     @Transactional
-    public boolean checkQudrigaAdmin(String userName)
-            throws QuadrigaStorageException {
+    public boolean checkQudrigaAdmin(String userName) throws QuadrigaStorageException {
```

Comment:
Since you're already touching this file, can we correct the spelling mistake in the method name?